Figured out how to make cool connect 4 cursor only show within board.

prra-codes opened the pull request:
Summary: The above. Adjusted CSS so that on board hover the regular cursor doesn't show, and that only when the mouse is over the board, the connect 4 cursor is shown.

Test Plan: Ran npm start and made sure I can win matches as usual. Working properly.

prra-codes committed:
Figured out how to make cool connect 4 cursor only show within board.

Summary: The above. Adjusted CSS so that on board hover the regular cursor doesn't show, and that only when the mouse is over the board, the connect 4 cursor is shown.

Test Plan: Ran npm start and made sure I can win matches as usual. Working properly.

prra-codes committed:
Added a server with express.js. Just says "Hello from server!"

Summary: Trying to learn and understand the basics of express.js. Installed express and produced a message to show "Hello from server!" that can run from port 8000.

Test Plan: Ran npm run dev to load nodemon server.js. when I got to localhost:8000/message, shows "Hello from server!"

prra-codes committed:
connecting front end and back end

summary: more trying to understand express.js. Got msg from backend "hello from server" to show in user interface.

test plan: ran npm start. works well.

jacksongabbard commented on client/src/components/Board/Board.css:
```diff
@@ -18,6 +14,27 @@
+}
+
+.board:hover + .cursor {
+  display: inline-block;
+  border-radius: 50%;
+  cursor: none;
```
👀

jacksongabbard reviewed (approved):
I'll approve this to unblock you, but when we next meet we *really* need to get you using a more effective Git workflow. A pull request with ~800 changed files + UI changes + server-side implementation is basically unreviewable. This isn't your fault -- you're still new at this. Progress, even if it's messy, is better than no progress. I'd always rather have on my team the person who will find a way to get work done as opposed to someone who only does clean/tidy work. Still, we gotta level you up a bit here. I'm sort-of back to healthy now so I reckon we can catch up next week properly.

prra-codes commented:
Got it. The one that says 800 changes files is literally just me moving all
my Connect 4 files into a folder called client, which I believe is why it
says 800 changed files. Thank you, I'm just trying to move forward with
everything in life right now! I'll make sure to learn the better methods we
speak about next week (hopefully you're well enough too)

On Thu, Jun 15, 2023 at 10:37 AM Jackson Gabbard ***@***.***>
wrote:

> ***@***.**** approved this pull request.
>
> I'll approve this to unblock you, but when we next meet we *really* need
> to get you using a more effective Git workflow. A pull request with ~800
> changed files + UI changes + server-side implementation is basically
> unreviewable. This isn't your fault -- you're still new at this. Progress,
> even if it's messy, is better than no progress. I'd always rather have on
> my team the person who will find a way to get work done as opposed to
> someone who only does clean/tidy work. Still, we gotta level you up a bit
> here. I'm sort-of back to healthy now so I reckon we can catch up next week
> properly.
> ------------------------------
>
> In client/src/components/Board/Board.css
> <https://github.com/prra-codes/peter-arthur-portfolio/pull/4#discussion_r1230728874>
> :
>
> > +  display: inline-block;
> +  border-radius: 50%;
> +  cursor: none;
>
> 👀
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/prra-codes/peter-arthur-portfolio/pull/4#pullrequestreview-1481131458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATTKUYC24KFL3476IJPG7FTXLLJU5ANCNFSM6AAAAAAY57QUTM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

prra-codes committed:
adjusted cursor

summary: made cursor even smaller on smaller screens. adjusted win screen text slightly.

test plan: ran npm start and inspected with google dev tools. looks good

prra-codes committed:
slight css correction
